Automattic/wp-calypso
# Checkout: Thank You: Update support section in footer with new copy

This pull request addresses part of https://github.com/Automattic/wp-calypso/issues/2484 by updating the footer of the `Thank You` page with a new copy for the support section:
 
![screenshot](https://cloud.githubusercontent.com/assets/594356/13495745/c760de2a-e14c-11e5-985f-15a4834d9728.png)
  
#### Testing instructions
 
1. Run `git checkout update/thank-you-footer` and start your server
2. Purchase any type of product
3. Check that the footer of the `Thank You` page displays as expected for [large viewports](https://cloud.githubusercontent.com/assets/12596797/10023359/b4ba5952-611f-11e5-9d47-0969f56faea0.png)
4. Check that the footer of the `Thank You` page displays as expected for [small viewports](https://cloud.githubusercontent.com/assets/12596797/9966352/6081dd0a-5e0a-11e5-917b-81f50e1dd087.png)
5. Purchase a Jetpack plan
6. Check that clicking on buttons now redirects you on the Jetpack site
7. Check that the support section displays correctly in the [app components doc](http://calypso.localhost:3000/devdocs/app-components/happiness-support)
 
#### Additional notes
 
I wasn't able to align vertically the gravatar, especially since there is a difference of placement for the buttons depending on the size of the viewport.
 
#### Reviews
